runner.rb

```diff
@@ -1,42 +1,70 @@
-require_relative 'account'
-require_relative 'user'
+require_relative 'bank'
 
-def make_account(name)
-  Account.new({ owner: name })
-end
+# SauceBank runner class
+class Runner
+  attr_accessor :customer
+  def initialize
+    @bank = Bank.new
+    @bank.create_account('latoya')
+    @magic_word = ''
+    welcome
+  end
 
-def print_menu(customer)
-  puts <<-EOT
-    #{customer.owner.name},
-    would you like to:
-    - see your balance, (balance)
-    - make a withdrawal, (withdraw) or
-    - make a deposit? (deposit)
-  EOT
-end
+  def confirm_customer(name)
+    if @bank.has_account?(name)
+      @customer = find_account(name).first
+    else
+      puts "looks like you don't have an account, No Worries, I'll make one!"
+      @customer = @bank.create_account(name)
+    end
+  end
 
-def goodbye
-  puts 'Goodbye, thanks for banking with SauceBank! Stay Saucey.'
-end
+  def print_menu
+    puts <<-EOT
+      #{customer.owner.name},
```

Comment:
What is a `customer.owner`?

Reply:
I wanted to differentiate the viewpoint of the `Bank`'s relationship to `User` and a `User`'s relationship as an owner of an `Account`. 

in this case, it's referring to the "customer" running the program at the time. 

I think I will look at my classes and make sure I'm not using too many things.